.templates/deconz/service.yml

```diff
@@ -0,0 +1,26 @@
+  deconz:
+    image: marthoc/deconz
+    container_name: deconz
+    restart: unless-stopped
+    network_mode: bridge
+    ports:
+      - '8090:80'
+      - '4439:443'
+      - '5901:5900'
+    volumes:
+       - ./volumes/deconz/:/root/.local/share/dresden-elektronik/deCONZ
+    devices:
```

Comment:
Gotta comment out this line too, or it'll produce this error when running the docker up command:
```
[2020-04-30T02:25:35+0100] [pi@raspberrypi] IOTstack (pr/24) $ docker-compose up -d
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.deconz.devices contains an invalid type, it should be an array
```

Reply:
sry. devices commented out now